src/main.rs

```diff
@@ -157,7 +164,7 @@ fn main() -> std::result::Result<(), Box<dyn std::error::Error + 'static>> {
         xs
     } else {
         initialize_package_json(&flags.destination)
-            .context("Failed to use npm to initialize the repo.")?;
+            .context(format!("Failed to run `npm init -y` in {:?}", flags.destination))?;
```

Comment:
We may want to use `with_context` here. It takes a lambda so we don't allocate a string unless we hit the error case. (It's not going to break the performance bank here, as you know, but it might be worth starting to build callus around.)